### PR TITLE
Update RIF samples for header row, date format, and bene IDs

### PR DESCRIPTION
**Pull request by karlmdavis**
This covers the work from [CBBD-99: Update sample data sets to latest, fix their naming, ensure proper grouping](http://issues.hhsdevcloud.us/browse/CBBD-99).

### CODE REVIEW

**Comment by karlmdavis** [comment]
Oh, forgot to mention this in my commit message: I also fixed up the claim/event files' bene IDs to match in `sample-a-*`.

**Comment by karlmdavis** [comment]
@sbrockhoff and/or @dho937: I'm going to merge this one in *before* review, as the bene ID fix in it is blocking me on another task. Would you please review after the fact, though?

We're going to need another dump from Scott K anyways, as the claim lines are still not correctly grouped. I'll handle that in a later pull request, though.

File: bluebutton-data-pipeline-sampledata/src/main/java/gov/hhs/cms/bluebutton/datapipeline/sampledata/StaticRifResource.java [hunk 1]
```diff
@@ -6,22 +6,37 @@
...
+	 * This file was manually created by copying a single claim from
+	 * {@link StaticRifResource#SAMPLE_B_CARRIER}, and adjusting its beneficiary
+	 * to match {@link #SAMPLE_A_BENES}.
+	 */
+	SAMPLE_A_CARRIER("rif-static-samples/sample-a-bcarrier.txt", RifFileType.CARRIER, 1),
 
-	PDE_1195("rif-static-samples/pde-1195.txt", RifFileType.PDE, 1195),
+	/**
+	 * This file was manually created by copying a single claim from
+	 * {@link StaticRifResource#SAMPLE_B_CARRIER}, adjusting its beneficiary to
```
Comment (sbrockhoff) [inline bluebutton-data-pipeline-sampledata/src/main/java/gov/hhs/cms/bluebutton/datapipeline/sampledata/StaticRifResource.java]: I believe you mean from SAMPLE_B_PDE here?

File: bluebutton-data-pipeline-sampledata/src/main/resources/rif-static-samples/readme.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+These files are effectively just comma-separated value (CSV) files, except the field delimiter used is `|` instead of a comma. There are a number of useful Unix tools that can be used to directly analyze such files.
+
+For analyzing a single file, it's probably simplest to use `awk`. As an example, the following command can be used to calculate the number of distinct `CLM_ID` (the 4th column) values in the `bcarrier.txt` file:
+
+    $ awk -F '|' '{print $4}' bcarrier.txt | tail -n +2 | sort | uniq -c | wc -l
+
+If one needs to run a query across multiple files, it's probably best to use a more sophisticated tool. I've found [the q tool](http://harelba.github.io/q/) to be quite useful: it allows you to run any SQL query against any set of CSV data. For example, I used the following query to determine that there is no single beneficiary in the `sample-b-*.txt` files that has a claim of every type (only took 1.3s to run):
+
+    $ q --skip-header --delimiter="|" \
```
Comment (sbrockhoff) [inline bluebutton-data-pipeline-sampledata/src/main/resources/rif-static-samples/readme.md]: No feedback, this is just pretty cool!

**Comment by sbrockhoff** [comment]
:+1:

**Pull request merged**